src/item.cpp

```diff
@@ -4318,21 +4322,8 @@ int item::get_encumber_when_containing(
             break;
     }
 
-    const int thickness = get_thickness();
-    const int coverage = get_coverage();
-    if( item_tags.count( "wooled" ) ) {
-        encumber += 1 + 3 * coverage / 100;
-    }
-    if( item_tags.count( "furred" ) ) {
-        encumber += 1 + 4 * coverage / 100;
-    }
-
-    if( item_tags.count( "leather_padded" ) ) {
-        encumber += ceil( 2 * thickness * coverage / 100.0f );
-    }
-    if( item_tags.count( "kevlar_padded" ) ) {
-        encumber += ceil( 2 * thickness * coverage / 100.0f );
-    }
+    encumber += std::max( 1, static_cast<int>( ceil( get_clothing_mod_val(
```

Comment:
If I'm not mistaken, this is adding an extra point of encumbrance to everything, and shouldn't be. Probably why the tests are failing?

Reply:
you are right. fixed.